stories/8-Toast.stories.tsx

```diff
@@ -0,0 +1,96 @@
+import "./utils/constants/toast/_toast-story-sample.scss";
+
+import {storiesOf} from "@storybook/react";
+import React from "react";
+
+import Button from "../src/button/Button";
+import {useToaster} from "../src/toast/util/toastHooks";
+import StoryFragment from "./utils/StoryFragment";
+
+function ToastComponent() {
+  const {display, update, hideAll} = useToaster();
+
+  return (
+    <StoryFragment>
+      <div className={"toast-button-group"}>
+        <Button
+          type={"button"}
+          onClick={() =>
+            display({
+              mode: "info",
+              content: "Information Message"
+            })
+          }>
+          {"Info Toast"}
+        </Button>
+
+        <Button
+          type={"button"}
+          onClick={() =>
+            display({
+              mode: "success",
+              content: "Success Message"
+            })
+          }>
+          {"Success Toast"}
+        </Button>
+
+        <Button
+          type={"button"}
+          onClick={() =>
+            display({
+              mode: "warning",
+              content: "Warning Message"
+            })
+          }>
+          {"Warning Toast"}
+        </Button>
+
+        <Button
+          type={"button"}
+          onClick={() =>
+            display({
+              mode: "danger",
+              content: "Error Message"
+            })
+          }>
+          {"Error Toast"}
+        </Button>
+      </div>
+
+      <br />
+
+      <div className={"toast-button-group"}>
+        <Button
+          type={"button"}
+          onClick={() =>
+            display({
+              mode: "info",
+              content: "Info Toast with Custom ID",
+              id: "custom-info-toast"
+            })
+          }>
+          {"Info Toast with Custom ID"}
+        </Button>
+
+        <Button
+          type={"button"}
+          onClick={() =>
+            update("custom-info-toast", {
+              mode: "success",
+              content: "Custom Toast has updated!",
+              timeout: 10000
+            })
+          }>
+          {"Update Custom Toast"}
+        </Button>
+
+        <Button type={"button"} onClick={hideAll}>
+          {"Hide All"}
+        </Button>
+      </div>
```

Comment:
Let's add an example where the toast doesn't `autoClose` but it is controlled by a button within the provided "render" function to the toast.display function. I think the way we will have this functionality is to have a `ToastCloseButton` component which as access to that particular Toast's id so that it can close it. This will be another compound components pattern. Each Toast component should provide a context to their children and this ToastCloseButton can then access this context to use Toast's id. We will export ToastCloseButton from ui-toolkit as well so that it can be used within projects.

Reply:
I've added an example and `ToastStory` component to use compound component pattern. Should we create like that?

Reply:
No. See my comment below. We need to provide this functionality to the users of this toolkit